docs/guide/rpc-api.md

```diff
@@ -29,14 +29,14 @@ Important methods from this API include:
 - [`eth_sendTransaction`](https://eth.wiki/json-rpc/API#eth_sendtransaction)
 - [`eth_sign`](https://eth.wiki/json-rpc/API#eth_sign)
 
-## Permissions
+## Restricted Methods
 
 MetaMask introduced Web3 Wallet Permissions via [EIP-2255](https://eips.ethereum.org/EIPS/eip-2255).
-In this permissions system, each RPC method is either _restricted_ or _open_.
-If a method is restricted, an external _domain_ (like a web3 site) must have the corresponding permission in order to call it.
-Open methods, meanwhile, do not require permissions to call, but may require confirmation by the user in order to succeed (e.g. `eth_sendTransaction`).
+In this permissions system, MetaMask is the _host_ environment, and all of its RPC methods are either _restricted_ or _unrestricted_.
+If a method is restricted, an external _subject_ (like a web3 site) must have the corresponding permission in order to call it.
+Unrestricted methods, meanwhile, do not require permissions to call, but may require confirmation by the user in order to succeed (e.g. `eth_sendTransaction`).
```

Comment:
`eth_sendTransaction` is not a great example of this because it's a restricted method. It requires `eth_accounts`.

Reply:
```suggestion
Unrestricted methods, meanwhile, do not require permissions to call, but may require confirmation by the user in order to succeed (e.g. `wallet_addEthereumChain`).
```

Reply:
Addressed in https://github.com/MetaMask/metamask-docs/pull/342